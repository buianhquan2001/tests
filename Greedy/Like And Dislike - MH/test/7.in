10
24 62 44
26 34 58
17 11 81
1 6 0
80 100 86
48 21 60
26 35 61
84 64 75
26 89 95
34 90 85
