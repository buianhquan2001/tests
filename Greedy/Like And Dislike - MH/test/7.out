Unknown
Unknown
Unknown
Dislike
Unknown
Unknown
Unknown
Unknown
Unknown
Unknown
